observing/obsstate.py

```diff
@@ -124,24 +142,23 @@ def add_session(sdffile: str):
                    session.STATUS))
 
 
-def add_settings(filename: str, time_loaded: str):
+def add_settings(filename: str):
     """Add settings to the database.
 
     Parameters
     ----------
     filename : str
         Name of the settings file.
-    time_loaded : str
-        Time the settings file was loaded.
     """
 
     assert os.path.exists(filename), f"{filename} does not exist"
     user = getpass.getuser()
+    time_loaded = Time.now().mjd
```

Comment:
Does python sqllite do the datetime object? This is pedantic but I don't like mjd/jd as a stored timestamp:

- It doesn't do leap seconds.
- It's stored as a floating point number -- bits are wasted in the exponent.

unix timestamp is an integer and so fixed precision. I usually prefer that. But typically databases do the conversions for you if you tell it that it's a datetime

Reply:
I do see docs on that in sqlite and I agree with your point. I'll punt on it for now, but try to improve timestamp management.